packages/context-menu/src/vaadin-menu-overlay-mixin.js

```diff
@@ -3,13 +3,15 @@
  * Copyright (c) 2016 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
+import { getClosestElement } from '@vaadin/component-base/src/dom-utils.js';
+import { OverlayFocusMixin } from '@vaadin/overlay/src/vaadin-overlay-focus-mixin.js';
 import { PositionMixin } from '@vaadin/overlay/src/vaadin-overlay-position-mixin.js';
 
 /**
  * @polymerMixin
  */
 export const MenuOverlayMixin = (superClass) =>
-  class MenuOverlayMixin extends PositionMixin(superClass) {
+  class MenuOverlayMixin extends OverlayFocusMixin(PositionMixin(superClass)) {
```

Comment:
Adding `OverlayFocusMixin` here shouldn't be needed because `Overlay` already applies it.

Reply:
This was added because [of this question](https://github.com/vaadin/web-components/pull/5881#discussion_r1203708336) I made earlier:

Reply:
I did it primarily for the sake of clarity. This way it becomes clearer why we need to override `_deepContains` and `_shouldRestoreFocus`.

Reply:
Thanks, fair enough.